src/components/pages/Stake/reducer.ts

```diff
@@ -146,28 +200,41 @@ const calculate = (state: State): State => {
       currentAPY: apy.apy,
       poolShare: apy.share,
     };
-  } else if (
+  }
+  if (
```

Comment:
Wait, do we need to change the simulation at all? :thinking: Not sure I understand this.